text/0000-Project-Goals-2024h2.md

```diff
@@ -0,0 +1,522 @@
+| *Inside Rust blog post inviting feedback*                                                             |                      |                                                                                           |
+| ↳ [Assemble project goal slate](https://rust-lang.github.io/rust-project-goals/2024h2/Project-goal-slate.html#ownership-and-team-asks)                        | [Niko Matsakis][]        | [Posted](https://blog.rust-lang.org/inside-rust/2024/05/07/announcing-project-goals.html) |
+| *RFC decision*                                                                                        |                      |                                                                                           |
+| ↳ [Assemble project goal slate](https://rust-lang.github.io/rust-project-goals/2024h2/Project-goal-slate.html#ownership-and-team-asks)                        | [Niko Matsakis][]        | ![Complete][]                                                                             |
+| *Top-level Rust blog post*                                                                            |                      |                                                                                           |
+| ↳ [Rust 2024 Edition](https://rust-lang.github.io/rust-project-goals/2024h2/Rust-2024-Edition.html#ownership-and-team-asks)                                   | [TC][]         |                                                                                           |
+| *Top-level Rust blog post announcing result*                                                          |                      |                                                                                           |
+| ↳ [Assemble project goal slate](https://rust-lang.github.io/rust-project-goals/2024h2/Project-goal-slate.html#ownership-and-team-asks)                        | [Niko Matsakis][]        |                                                                                           |
+| *Top-level Rust blog post inviting feedback*                                                          |                      |                                                                                           |
+| ↳ [Make Rustdoc Search easier to learn](https://rust-lang.github.io/rust-project-goals/2024h2/rustdoc-search.html#ownership-and-team-asks)                    | [Michael Howell][]           |                                                                                           |
+| ↳ [Use annotate-snippets for rustc diagnostic output](https://rust-lang.github.io/rust-project-goals/2024h2/annotate-snippets.html#ownership-and-team-asks)   | [Esteban Kuber][], [Scott Schafer][] |                                                                                           |
```

Comment:
As above, I don't think these are really under the council. At one point in time, core did approve posts to the main blog, but I think it's been a while since that requirement (the council surely hasn't been approving any).